### DISCUSSION TITLE
Does chia rpc 8555 can no longer be call using curl ?

### DISCUSSION THREAD

**Discussion by wiryonolau** [Chia-Network/chia-blockchain#18502] [Support]
Hi after updating to 2.4.2 all call to 8555 using curl ( http api ) ?
any change in this feature ?

Always got connection refuse 
If call locally got 500 Internal Server Error

```
curl -H "Content-Type: application/json" -X POST -k https://127.0.0.1:8555/get_blockchain_state --cert private_full_node.crt --key private_full_node.key

500 Internal Server Error
```

Which cert and key should I use for the request, previously it always use private_full_node certificate and key

**Comment by BrandtH22** [comment]
Hey @wiryonolau , it looks like no request data has been added to the command. Try to run that again with `-d "{}"` added:

```
curl -H "Content-Type: application/json" -X POST -k https://127.0.0.1:8555/get_blockchain_state -d "{}" --cert private_full_node.crt --key private_full_node.key
```

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
